README.md

```diff
@@ -98,6 +98,13 @@ You can find the documentations in the [`docs`](./docs) folder.
 * [`encodeURIComponent`](./docs/string.md#encodeuricomponent)
 * [`repeat`](./docs/string.md#repeat)
 * [`truncate`](./docs/string.md#truncate)
+* [`slugify`](./docs/string.md#slugify)
+* [`endsWith`](./docs/string.md#endsWith)
+* [`stripTags`](./docs/string.md#stripTags)
```

Comment:
The markdown link after the # character does not take uppercase character. Write it all in lowercase.